rwatch/ui/layer/single_notification_layer.c

```diff
@@ -161,81 +169,61 @@ uint16_t single_notification_layer_height(SingleNotificationLayer *l) {
         l->timestamp, fonts_get_system_font(TIMESTAMP_FONT),
         szrect, GTextOverflowModeTrailingEllipsis, GTextAlignmentLeft, 0).h
         + ELEMENT_PADDING;
-    
+
     return height;
 }
 
 static void single_notification_layer_update_proc(Layer *layer, GContext *ctx) {
     SingleNotificationLayer *l = container_of(layer, SingleNotificationLayer, layer);
     GRect szrect = layer_get_frame(layer);
     GSize outsz;
-    
+
     graphics_context_set_fill_color(ctx, GColorWhite);
     graphics_fill_rect(ctx, szrect, 0, GCornerNone);
-    
+
     graphics_context_set_text_color(ctx, GColorBlack);
-    
+
     szrect.origin.x = 0;
-    szrect.origin.y = 0;
-    
+    szrect.origin.y = STATUS_BAR_LAYER_HEIGHT;
```

Comment:
Hmm, wait, is this correct when notifications start scrolling?  I think this needs to be 0, and the offset should be specified in notification_window, right?  Otherwise, the secondary notification that you scroll up into will start too low on the screen?